commands/milestone_report_command.go

```diff
@@ -0,0 +1,61 @@
+package commands
+
+import (
+	"context"
+	"fmt"
+	"github.com/hanochg/piperika/http"
+	"github.com/hanochg/piperika/report"
+	"github.com/hanochg/piperika/utils"
+	"github.com/jfrog/jfrog-cli-core/plugins/components"
+	"sync"
+	"time"
+)
+
+const defaultBranchName string = "master"
+
+var services = []string{"access", "metadata", "event", "router", "jfconnect", "mission_control", "integration"}
```

Comment:
Please move the constants\config to the utils folder (you can create a new file called config.go if needed)